hw/ip/rstmgr/data/rstmgr.hjson

```diff
@@ -0,0 +1,172 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+// TODO: This module is only a draft implementation that covers most of the rstmgr
+// functoinality but is incomplete
+
+# RSTMGR register template
+#
+{
+  name: "RSTMGR",
+  clock_primary: "clk_i",
+  other_clock_list: [
+    "clk_main_i",
+    "clk_fixed_i",
+    "clk_usb_i"
+  ],
+  bus_device: "tlul",
+  regwidth: "32",
+  param_list: [
```

Comment:
@vogelpi 
o i see what you were saying. Yes I actually meant to remove these (see other PR). Sorry got lost when I was shifting and moving files around.